cmd/internal/root.go

```diff
@@ -132,14 +137,20 @@ func init() {
 
 // newLogger returns a new zap logger
 func newLogger(options ...zap.Option) *zap.SugaredLogger {
-
-	logger, err := zap.NewDevelopment(options...)
+	level := viper.GetString("log-level")
+	l := zap.DebugLevel
+	l.Set(level)
+
+	cfg := zap.NewDevelopmentConfig()
```

Comment:
Why we don't use `NewProductionConfig` directly if we don't want to keep development env? Do we have customized something?

Reply:
It's very different from the log format we have now: https://github.com/uber-go/zap/blob/master/config.go#L114-L127
Judging from the code it outputs everything as JSON.
I tried it a few months back and if I remember correctly, it doesn't contain code locations.

Setting `Development: false` just makes the stack traces more dense, according to documentation.